package.json

```diff
@@ -60,12 +60,13 @@
         "bs58check": "^4.0.0",
         "buffer": "^6.0.3",
         "eventemitter3": "^5.0.1",
-        "lodash.isequal": "^4.5.0"
+        "lodash.isequal": "^4.5.0",
+        "path": "^0.12.7"
```

Comment:
Its only used by tests.
move this in devDependencies

Reply:
are you sure is needed?
path is included in node standard library. and the e2e tests are run in "node" environement in jest. so plz confirm this